docs/getting-started-auditor.md

```diff
@@ -0,0 +1,170 @@
+However, for this getting started guide, we assume they are placed in the same network and managed in the same administrative domain.
+
+
+## Assumptions
+
+In this guide, we assume Ledger and Auditor both use Cassandra through Scalar DB,
+both Cassandra instances use default username and password for the admin privilege.
+Also, Ledger, Auditor, and Cassandra are all located in the same network so that they can access each other.
+
+## Configure properties
+
+You need to configure Ledger and Auditor to make the detection properly work.
+
+For the properties of Ledger, you need to configure the following entries:
+```
+[ledger.properties]
+scalar.dl.ledger.proof.enabled=true
+scalar.dl.ledger.proof.private_key_path=/path/to/private_key.pem
+scalar.dl.ledger.nonce_txid.enabled=true
+scalar.dl.ledger.auditor.enabled=true
+#scalar.dl.ledger.auditor.cert_holder_id=auditor
+#scalar.dl.ledger.auditor.cert_version=1
+```
+
+You first need to set `scalar.dl.ledger.auditor.enabled` to true to let Ledger know if the Scalar DL system uses Auditor.
+Note that you also need to enable `scalar.dl.ledger.proof.enabled` and set a proper private key to `scalar.dl.ledger.proof.private_key_path` since Auditor uses [AssetProof](how-to-use-proof.md) to work.
+If they are not properly and consistently configured, Ledger will throw an exception.
+
+Also, Ledger manages an Auditor's certificate (i.e., Auditor needs to register its certificate to Ledger) and uses the certificate to validate a request signed by Auditor's private key to identify the request origin; thus, the certificate holder ID (`scalar.dl.ledger.auditor.cert_holder_id`) and version (`scalar.dl.ledger.auditor.cert_version`) are required to be set as well. 
+By default, Ledger assumes Auditor registers its certificate with a name `auditor` and version `1`.
+
+For the properties of Auditor, you need to configure the following entries:
+```
+[auditor.properties]
+#scalar.dl.auditor.ledger.cert_holder_id=ledger
+#scalar.dl.auditor.ledger.cert_version=1
+#scalar.dl.auditor.cert_holder_id=auditor
+#scalar.dl.auditor.cert_version=1
+scalar.dl.auditor.cert_path=/path/to/auditor.pem
+scalar.dl.auditor.private_key_path=/path/to/auditor-key.pem
+```
+
+Auditor manages a key pair to sign a request before sending the request to Ledger and validate a request given from Ledger,
+so you need to set `scalar.dl.auditor.cert_path` and `scalar.dl.auditor.private_key_path` properly.
+
+As similarly to Ledger, Auditor manages a Ledger's certificate (i.e., Ledger registers its certificate to Auditor) and uses the certificate to validate a request signed by Ledger's private key; thus, the certificate holder ID (`scalar.dl.auditor.ledger.cert_holder_id=ledger`) and version (`scalar.dl.auditor.ledger.cert_version`) are required.
+By default, Auditor assumes Ledger registers its certificate with a name `ledger` and version `1`.
+
+Other values are optional when you but they need to be updated depending on an environment.
```

Comment:
This sentence is a bit unclear. I think some word is missing here? "Other values are optional when you <something> but they need to be updated depending on an environment"

Reply:
Thank you!
Fixed in e493635c1286d7822e3d13175b75abdd0fd2cd61.